### DISCUSSION TITLE
stuck focus bar

### DISCUSSION THREAD

**Discussion by dmitrovich** [mucommander/mucommander#568] [Q&A]
Using muCommander 0.9.7-1 on macOS High Sierra X.13.6.  I have 2 muCommander windows open (a total of 4 panes, each a different folder).  Occasionally on one of the panes of one of the windows, the focus bar that usually scrolls up and down to select a file, gets stuck near the middle.  Trying to move it up or down has it jump back to its original stuck position.  Often I have to quit that window and re-open it to get proper behavior.
Any idea what might cause that and how to prevent it/fix it?
Thanks.

**Comment by ahadas** [comment]
hi @dmitrovich , it didn't happen to me so I don't know - but it could be one of the things that will be sorted out in the upcoming version that would come with Java 17

**Comment by ahadas** [comment]
@dmitrovich v1.0.0 has been released, how about giving it a try?
